ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -679,12 +679,17 @@ objects:
           cat > /tmp/proxy.ign << EOF
         {
           "ignition": {
-            "config": {},
+            "config": {
```

Comment:
Nit: I think all the empty fields can be dropped here as ignition uses a parser implementation that allows for that

Other than that, I think this change should go in